riffdog/scanner.py

```diff
@@ -119,41 +132,99 @@ def _real_scan_element(scan_element, scanned, scanning):
     if element:
         for required_element in element.depends_on:
             if required_element not in scanned:
+                logger.info("Lookging for %s resources in real world" % required_element)
                 scanned = _real_scan_element(required_element, scanned, scanning)
 
         element.fetch_real_resources()
 
     scanned.append(scan_element)
     return scanned
 
+def _disk_inspector(state_location):
+    logger.info("File State Inspecting %s" % state_location)
+
+    target = state_location
+
+    if os.path.exists(target):
+        if os.path.isdir(target):
+            # get directories
+            for root, dirs, files in os.walk(target):
+                for filename in files:
+                    _file_inspector("%s%s" % (state_location,filename))
+
+                for direc in dirs:
+                    _disk_inspector("%s/%s/" % (state_location, direc))
+
+        if os.path.isfile(target):
+            _file_inspector(target)
+    else:
+        logger.warning("Location %s does not seem to exist" % state_location)
+    pass
```

Comment:
This is redundant

Reply:
I disagree, and to prove it here:
`(venv) MacBook-Pro-5:riffdog jt$ riffdog -v -i test_resource_pack  /tmp/not_a_place
03/07/2020 12:19:37 Exception loading riffdog_aws - might not be installed, or errors on load
03/07/2020 12:19:37 Location /tmp/not_a_place does not seem to exist`

Basically when dev-ing the file one, I was passing a typo in, and this warning saved me.

Reply:
(unless you meant the pass, which I've chopped)

Reply:
Yup I meant the pass

Reply:
Ok, done XD